### ISSUE TITLE
Bug: Angular ESLint Flagging `@angular-eslint/template/no-duplication-attributes` for Valid Scenario

### ISSUE THREAD

**Issue by wizardnet972** [angular-eslint/angular-eslint#1948]
This scenario is valid in Angular. Angular maintains the `icon` class and allows for the addition of another class using the `icon` variable. 

```html
<i class="icon" [class]="icon"></i>
```

The Angular ESLint rule `@angular-eslint/template/no-duplication-attributes` identifies this as a duplication of attributes.

**Comment by wizardnet972** [comment]
workaround: `<i class="icon" [ngClass]="icon"></i>`

**Comment by json-derulo** [comment]
I think this is working as intended, see the [rule doc](https://github.com/angular-eslint/angular-eslint/blob/103df48740e1601ad0c6a3979c87153cb4ab3de5/packages/eslint-plugin-template/docs/rules/no-duplicate-attributes.md). With the following configuration you can ignore the errors specifically for the `class` attribute:

```
{
  "rules": {
    "@angular-eslint/template/no-duplicate-attributes": [
      "error",
      {
        "ignore": [
          "class"
        ]
      }
    ]
  }
}
```
